spec/requests/old_events_spec.rb

```diff
@@ -915,7 +915,10 @@
   end
 
   context "show" do
-    let(:doi) { create(:doi, client: client, aasm_state: "findable") }
+    let(:doi) do
+      allow_any_instance_of(DataciteDoi).to receive(:send_import_message)
```

Comment:
Does this intercept the message in tests so we can remove the guard in the Indexable concern?

Reply:
I am not sure if I get your question, but previously we never used to execute `send_import_message` method in test env, but I have removed that condition as I wanted to write a code which checks when we should execute that method.

Hence we need to mock this method call.

https://github.com/datacite/lupo/blob/master/app/models/concerns/indexable.rb#L32

Let me know if I answered your query :)

Reply:
Cool